### DISCUSSION TITLE
Tested Stacks: Add information about the computer you have tested this on here.

### DISCUSSION THREAD

**Discussion by w4ffl35** [w4ffl35/krita_stable_diffusion#16] [General]
If you tested this plugin, please let me know the results and information about the computer you tested it on, including:

- **Operating System** - Windows 11, Ubuntu 20.04 etc
- **GPU**
- **CPU**
- **Krita version**

If you change anything like cuda drivers etc. please list that as well

**Comment by ACMOIDRE** [comment]
Does it support and how long it take to generate 1 image on
win 10
gtx 1650 Ti
cpu i5 11th
Krita 5
